langDir/fr.json

```diff
@@ -141,18 +141,18 @@
         "l1": "[Restez avec nous et suivez-nous @KodaDot !](@.common:links.kodadot_social.twitter.href)"
       },
       "15": {
-        "m": "Nous sommes en train de mettre en place une vue de la collection, des ensembles de vues de la collection. @.md:faq.a.14.l1"
+        "m": "Nous sommes en train de mettre en place une vue de la collection, des ensembles de vues de collections. @.md:faq.a.14.l1"
       },
       "16": {
         "m": "C'est ce cœur 🖤 sous le jeton NFT. Nous sommes @.md:faq.a.16.l1 qui fonctionne comme des appréciations. Sa fonction est de découvrir le prix de votre travail.",
         "l1": "[auteurs d'origine d'EMOTE au standard RMRK](@.common:links.rmrk_spec_emote_issue.href)"
       },
       "17": {
-        "m": "En ce moment, nous avons @.md:faq.a.17.l1 Nous recherchons activement des options de financement, nous pourrions éventuellement lever des fonds privés. DMs nous, si vous êtes intéressés par une vente privée.",
+        "m": "En ce moment, nous avons @.md:faq.a.17.l1 Nous recherchons activement des options de financement, nous pourrions éventuellement lever des fonds privés. Envoyez-nous un MD, si vous êtes intéressés par une vente privée.",
```

Comment:
```suggestion
        "m": "En ce moment, nous avons @.md:faq.a.17.l1 Nous recherchons activement des options de financement, nous pourrions éventuellement lever des fonds privés. Envoyez-nous un DM, si vous êtes intéressés par une vente privée.",
```
DM fit better

Reply:
both would work, although as mentioned in previously, MD is the actual way to shortcut "message direct" in French.

Reply:
As @roiLeo suggested, MP is the best option.